.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+Make sure you have checked _all_ steps below.
+
+### Jira
+
+- [ ] My PR addresses the following [Parquet Jira](https://issues.apache.org/jira/browse/PARQUET/) issues and references them in the PR title. For example, "PARQUET-1234 My Parquet PR"
```

Comment:
We use the format "PARQUET-1234: My Parquet PR", i.e., there is a colon after the JIRA number.

Reply:
Thanks for clearing this up, I see mixed styles being used.